tests/unit/s2n_drbg_test.c

```diff
@@ -295,6 +295,10 @@ int check_drgb_version(s2n_drbg_mode mode, int (*generator)(void *, uint32_t), i
         /* Read the next personalization string */
         POSIX_GUARD(s2n_stuffer_read(&personalization, &personalization_string));
 
+        /* s2n_rand_set_callbacks overrode the default callbacks without cleaning up the default callbacks.
+        * Call defaults rand cleanup function to cleanup old callbacks before setting new ones. */
+        POSIX_GUARD_RESULT(s2n_rand_cleanup());
```

Comment:
This comment is no longer accurate. You could probably fix it by just switching "overrode" to "overrides", but this change is also much more straightforward now so a simpler comment is sufficient. The fact that the callbacks are the default callbacks is no longer important.
```suggestion
        /* Cleanup the existing callbacks before setting new callbacks */
        POSIX_GUARD_RESULT(s2n_rand_cleanup());
```